CRM/Contact/Form/Inline.php

```diff
@@ -77,11 +78,28 @@ public function preProcess() {
     $this->setAction(CRM_Core_Action::UPDATE);
   }
 
+  /**
+   * Get the contact ID.
+   *
+   * Override this for more complex retrieval as required by the form.
+   *
+   * @return int|null
+   *
+   * @noinspection PhpUnhandledExceptionInspection
+   * @noinspection PhpDocMissingThrowsInspection
+   */
+  public function getContactID(): int {
```

Comment:
I would have called the function `getContactId`... 
dunno if we'll ever all agree on that spelling.